src/main/java/no/ssb/subsetsservice/SubsetsController.java

```diff
@@ -62,6 +69,175 @@ public SubsetsController(){
         return getFrom(LDS_SUBSET_API, "/"+id);
     }
 
+    @GetMapping("/v1/subsets/{id}/versions")
+    public ResponseEntity<String> getVersions(@PathVariable("id") String id) {
+        return getFrom(LDS_SUBSET_API, "/"+id+"?timeline");
+    }
+
+    /**
+     * Get a list of versions of a subset that starts with {versions}.
+     * So if {versions} is "1.0", then 1.0.0, 1.0.1, 1.0.2 etc will be returned.
```

Comment:
I would discuss this. 
Patches are something users do not need to know about. For now, only the latest patched version is actually used on the client-side. 
I suggest to restrict `/versions/{version}` to respond with the requested version only.
We can later introduce `subset/{id}/history?filter={filters}` service if needed.

Reply:
Ok, I'll make it return only the given version (and not in a list) in the next update, then. PS: Since only the latest patched version is actually used on the client side, then this feature (get specific version) is not needed at all.